data/json/recipes/recipe_electronics.json

```diff
@@ -1983,5 +1983,21 @@
     [ [ "amplifier", 1 ] ],
     [ [ "cable", 2 ] ]
   ]
+},{
+  "type" : "recipe",
+  "result": "electrolysis_kit",
+  "category": "CC_ELECTRONIC",
+  "subcategory": "CSC_ELECTRONIC_TOOLS",
+  "skill_used": "electronics",
+  "using": [ [ "soldering_standard", 10 ] ],
+  "difficulty": 2,
+  "time": 10000,
+  "reversible": true,
+  "autolearn": true,
```

Comment:
Maybe it's better to add this recipe to some book? I feel the electrolysis is rather complicated process to figure it on its own.

Reply:
"Electrolysis kit" in this case is just a fancy name for a couple copper wires hooked up to improvised steel electrodes. Not your typical complex lab setup, any dabbling electrician could figure it out. I think the other recipe might warrant that though, stripping tin by combination of acid + electrolysis is pretty obscure knowledge. Any suggestion for which book?

Reply:
The tin stripping recipe is more of a chemistry, or of an electronics?

Reply:
DIY compendium sounds OK.